libraries/chain/apply_context.cpp

```diff
@@ -977,7 +928,7 @@ void apply_context::db_update_i64( int iterator, account_name payer, const char*
    });
 }
 
-void apply_context::db_remove_i64( int iterator ) {
+void apply_context::db_remove_i64_chainbase( int iterator ) {
```

Comment:
Initially, it was to make sure that I found all the macros in eosio-tester that were calling it (and added "_chainbase" to make it obvious this was the chainbase implementation).  I left it thinking it would remind me to ask what we wanted to do with this.  I'd like to pull this out into db_context_chainbase, but that won't happen till after next week there are just too many other things to do right at this moment.